src/lib/Scenes/Sale/Sale.tsx

```diff
@@ -48,29 +51,45 @@ export const Sale: React.FC<Props> = ({ queryRes }) => {
 
   const [isArtworksGridVisible, setArtworksGridVisible] = useState(false)
   const [isFilterArtworksModalVisible, setFilterArtworkModalVisible] = useState(false)
+  const [isLive, setIsLive] = useState(false)
+  const prevIsLive = usePrevious(isLive, false)
 
   const scrollAnim = useRef(new Animated.Value(0)).current
 
-  let intervalId: NodeJS.Timeout
-
-  useEffect(() => {
-    if (sale.liveStartAt) {
-      // poll every .5 seconds to check if sale has gone live
-      intervalId = setInterval(checkIfSaleIsLive, 500)
-      return () => {
-        clearInterval(intervalId)
-      }
+  // poll every .5 seconds to check if sale has gone live
+  useInterval(() => {
+    const now = moment()
+    if (sale.liveStartAt === null) {
```

Comment:
```suggestion
    if (sale.liveStartAt === null) {
      const now = moment()
```
I'll quickly do this. This feels slightly nicer.